src/Diagnostics.RuntimeHost/Startup.cs

```diff
@@ -160,6 +161,7 @@ public void ConfigureServices(IServiceCollection services)
             {
                 services.AddSingleton<IGithubClient, GithubClient>();
             }
+            services.AddSingleton<IDevOpsClient, DevOpsClient>();
```

Comment:
Shouldn't it be `IRepoClient` here instead of `IDevOpsClient`? 
`IDevOpsClient` seems to have only some properties defined but no methods.